Gruntfile.js

```diff
@@ -145,7 +146,7 @@ module.exports = function (grunt) {
                     },
                     "versions": {},
                     "updated": getFormattedDate(new Date(repoInfo.updated_at)),
-                    "downloads": Math.floor(Math.random() * 10000),
+                    "downloads": downloadStat[index] ?? 0,
```

Comment:
Do you want the default to be '0' or 'unavailable' or something in case the `got` fails? Not sure if you have hedges for failed `got`s elsewhere either, just curious about failure modes and debugging and such. :)

Reply:
if got fails then will be thrown an error and the process will be stopped. download statistics don't have a module if a module wasn't downloaded. for this reason, if a module is missing then 0

Reply:
check. :+1: :green_apple: